config/initializers/admin.rb

```diff
@@ -0,0 +1,7 @@
+require_relative "../../lib/restrict_admin_by_ip_middleware"
+
+Rails.application.configure do
+  allowed_ips = ENV.fetch("ADMIN_ALLOWED_IPS", "").split(",")
```

Comment:
Just to be clear, this defaults `allowed_ips` to an empty string if `ADMIN_ALLOWED_IPS` isn't set, which from my reading would mean that no one would be able to access admin (safe by default; good). Is this what we want or would it be better for the system to not boot and fail loudly if it the environment variable doesn't exist? Are there any environments where we expect it wouldn't be set?

Reply:
That's a good point. I'll make it fail, then.